src/app/admin/page.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { auth } from "@/auth";
+import { Payment, User, UserRole } from "@prisma/client";
+import { DataTable } from "./_components/datatable";
+import { columns } from "./_components/columns";
+
+async function getData() {
+  const response = await fetch("http://localhost:3000/api/users");
```

Comment:
For some reason, I need to use the absolute URL

Reply:
Now way! Can't u use `/api/users` ?

Reply:
Resolved in #[73ca97d](https://github.com/tiarasjec/website/pull/55/commits/73ca97dc744460017f308ef976e91b5f90f709f4)